src/api/json/catalog.json

```diff
@@ -110,7 +143,9 @@
       "url": "https://raw.githubusercontent.com/ansible-community/schemas/main/f/ansible-playbook.json",
       "fileMatch": [
         "playbook.yml",
-        "playbook.yaml"
+        "playbook.yaml",
```

Comment:
Should we have site.yaml and site.yml here?  I think they are pretty common, and suggested